spec/index.adoc

```diff
@@ -560,7 +560,15 @@ In order to call a canister, the user makes a POST request to `/api/v2/canister/
 * `method_name` (`text`): Name of the canister method to call
 * `arg` (`blob`): Argument to pass to the canister method
 
-The HTTP response to this request has an empty body and HTTP status 202, or a HTTP error (4xx or 5xx). Paranoid agents should not trust this response, and use <<http-read-state,`read_state`>> to determine the status of the call.
+The HTTP response to this request can have the following responses:
+
+* 202 HTTP status with empty body. Implying the request was accepted by the IC for further processing. Users should use <<http-read-state,`read_state`>> to determine the status of the call.
+* 200 HTTP status with non-empty body. Implying an execution pre-processing error occurred. The body of the response contains more information about the IC specific error encountered. The body is a CBOR map with the following fields:
+** `reject_code` (`nat`): The reject code (see <<reject-codes>>).
+** `reject_message` (`text`): a textual diagnostic message.
+** `error_code` (`text`): an optional implementation-specific textual error code (see <<error-codes>>).
+* 4xx HTTP status for client errors (e.g. malformed request or canister not found). Retrying the request will likely have the same outcome.
```

Comment:
> Retrying the request will likely have the same outcome.

I see http status codes like 429 Too Many Requests in the overview of http status codes which should be possible to retry. So I wonder we could rephrase this sentence, e.g., by listing the status codes that do not make sense to retry?

@rumenov

Reply:
I don't think it makes sense. Then we will be repeating the HTTP protocol standard.

Reply:
Sure, I don't want to repeat the HTTP protocol standard. Instead I want to explicitly list status codes produced by the IC due to IC-specific failures (canister not found is an example that likely has the same outcome unless the canister is created in the meantime as thus should be listed as a case to which the sentence "Retrying the request will likely have the same outcome" applies).

Reply:
Given this - https://github.com/dfinity/interface-spec/pull/143#discussion_r1151721927, there won't be any execution errors that map to 4xx/5xx errors.

Reply:
Perfect! I've made a suggestion to drop "canister not found" and refine the sentence on retrying.